website/docs/integrator-section/verifiable-document/ethereum/oa-embedded-html.md

```diff
@@ -0,0 +1,93 @@
+---
+id: oa-embedded-html
+title: Embed OA Doc in HTML File
+sidebar_label: Embed OA Doc in HTML File
+---
+
+Depending on your use case, after creating an issued/signed OA document, you may want to embed it into a HTML file for issuance.
+
+## How It Works
+
+In this guide, we will create a HTML file with a "Proceed" button to redirect the user to [verify.gov.sg](https://www.verify.gov.sg/). The OA document will be embedded in the Verify URL for verification and rendering.
+
+The steps are as follows:
+
+1. Create our Verify URL by encoding and embedding the OA document data at the back of the Verify URL
```

Comment:
Create the Verify URL

Reply:
Updated the line - https://github.com/Open-Attestation/documentation-website/pull/164/files#diff-3e1c2cbc47795d977506ec6522a012a9cd63a5d2c7a40a34df471811ab497faaR15